data_safe_haven/commands/config.py

```diff
@@ -51,6 +53,55 @@ def show_shm(
 
 
 # Commands related to an SRE
+@config_command_group.command()
+def available() -> None:
+    """List the available SRE configurations for the selected Data Safe Haven context"""
+    logger = get_logger()
+    try:
+        context = ContextManager.from_file().assert_context()
+    except DataSafeHavenConfigError as exc:
+        logger.critical(
+            "No context is selected. Use `dsh context add` to create a context "
+            "or `dsh context switch` to select one."
+        )
+        raise typer.Exit(1) from exc
+    azure_sdk = AzureSdk(context.subscription_name)
+    try:
+        blobs = azure_sdk.list_blobs(
+            container_name=context.storage_container_name,
+            prefix="sre",
+            resource_group_name=context.resource_group_name,
+            storage_account_name=context.storage_account_name,
+        )
+    except DataSafeHavenAzureStorageError as exc:
+        logger.critical("Ensure SHM is deployed before attempting to use SRE configs.")
+        raise typer.Exit(1) from exc
+    if not blobs:
+        logger.critical(f"No configurations found for context '{context.name}'.")
+        raise typer.Exit(0)
```

Comment:
```suggestion
        logger.info(f"No configurations found for context '{context.name}'.")
        raise typer.Exit(0)
```

I think let's make that an info message if it isn't a fatal error.